shared/actions/kbfs/index.desktop.js

```diff
@@ -4,8 +4,19 @@ import {shell} from 'electron'
 import * as Constants from '../../constants/config'
 import {config} from '../../constants/types/keybase-v1'
 import type {AsyncAction} from '../../constants/types/flux'
+import {cleanup} from '../../util/kbfs'
 
 const open = (kbfsPath: string, path: ?string) => {
```

Comment:
ensure we call cleanup on all paths that come in. the current cleanup assumes you don't pass the /private /public prefix so this'll strip that out then re-add it